lib/service/teams/listeners/message-notification-stream-listener.js

```diff
@@ -0,0 +1,7 @@
+function handler(event) {
+  console.log('You received a message notification! Ding!')
+}
```

Comment:
what is this for? how is this different than `new-message-listener`? is it purely for notifications?